app/javascript/packs/clock.js

```diff
@@ -53,28 +73,45 @@ function reset() {
   clearInterval(timerInterval);
   print("00:00:00");
   elapsedTime = 0;
+  lastLapTime = 0;
+  clearLaps();
   showButton("PLAY");
+  showButton("LAP");
+}
+
+function lap() {
+  let lapTime = elapsedTime - lastLapTime;
+  lastLapTime = elapsedTime;
+  printLap(timeToString(lapTime));
 }
 
 // Create function to display buttons
 
 function showButton(buttonKey) {
   if (buttonKey === "PLAY") {
-    playButton.classList.remove("d-none")
-    pauseButton.classList.remove("d-block")
-    pauseButton.classList.add("d-none")
+    playButton.classList.remove("d-none");
+    pauseButton.classList.remove("d-block");
+    pauseButton.classList.add("d-none");
```

Comment:
I think you could achieve the same result with just one line, but you would need to add all the classes that you need for that tag: `pauserButton.className('class-1 class-2')`

The same could apply for similar scenarios you have down below when you add and remove classes on the same tag

Reply:
Besides that, you did a good job! :rocket: :100:

Reply:
Thanks